### PR TITLE
en,zh: Bump Operator version to v1.2.6

### PR DESCRIPTION
**Pull request by just1900**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
Bump Operator to v1.2.6

Run commands:

```
find . -type f | xargs sed -i "s/v1.2.5/v1.2.6/g"
```

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: en/restart-a-tidb-cluster.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ spec:
 
 ## Perform a graceful restart to a single TiKV Pod
 
-Starting from v1.2.5, TiDB Operator supports graceful restart for a single TiKV Pod.
+Starting from v1.2.6, TiDB Operator supports graceful restart for a single TiKV Pod.
```
Comment (ran-huang) [inline en/restart-a-tidb-cluster.md:54]: 这个不用 bump 吧

Comment (DanielZhangQD) [reply en/restart-a-tidb-cluster.md:54]: 对，是

File: zh/restart-a-tidb-cluster.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ spec:
 
 ## 优雅重启单个 TiKV Pod
 
-从 v1.2.5 起，TiDB Operator 支持给 TiKV Pod 添加 annotation 来触发优雅重启单个 TiKV Pod。
+从 v1.2.6 起，TiDB Operator 支持给 TiKV Pod 添加 annotation 来触发优雅重启单个 TiKV Pod。
```
Comment (ran-huang) [inline zh/restart-a-tidb-cluster.md:44]: ？

File: en/restart-a-tidb-cluster.md (hunk 1 above)
Comment (DanielZhangQD) [reply en/restart-a-tidb-cluster.md:54]:
```suggestion
Starting from v1.2.5, TiDB Operator supports graceful restart for a single TiKV Pod.
```

File: zh/restart-a-tidb-cluster.md (hunk 2 above)
Comment (DanielZhangQD) [reply zh/restart-a-tidb-cluster.md:44]:
```suggestion
从 v1.2.5 起，TiDB Operator 支持给 TiKV Pod 添加 annotation 来触发优雅重启单个 TiKV Pod。
```

**Comment by DanielZhangQD** [comment]
@ran-huang PTAL again.

**Review by ran-huang — APPROVED** [review]

**Pull request merged**